interface.py

```diff
@@ -1,82 +0,0 @@
-import serial
```

Comment:
If at all possible I'd be in favour of keeping this wrapper. While in theory this firmware will mainly be used under [`robotd`](https://github.com/sourcebots/robotd), I've found this wrapper to be very useful for debugging things (both on the arduino and cases where `robotd` wasn't behaving as expected when talking to an arduino).

Reply:
I'd much rather build this into `robotd` if such an interface is required (reduce code replication). I'd also like to point out that a serial console is pretty useful for this purpose.

Reply:
Can we put the removal in a separate PR and debate this there?